doc/source/cluster/kubernetes/user-guides/rayservice-no-ray-serve-replica.md

```diff
@@ -0,0 +1,216 @@
+```
+:::
+
+::::
+
+Look at the Ray Serve configuration `serveConfigV2` embedded in the RayService YAML. Notice the only deployment in `deployments` of the application named `simple_app`:
+  * `num_replicas`: Controls the number of replicas to run that handle requests to this deployment. Initialize to 1 to ensure the overall number of the ray serve replicas is 1.
+  * `max_replicas_per_node`: Controls the maximum number of replicas on one single pod.  
+
+See [Ray Serve Documentation](https://docs.ray.io/en/master/serve/configure-serve-deployment.html) for more details.
+```yaml
+serveConfigV2: |
+  applications:
+    - name: simple_app
+      import_path: ray-operator.config.samples.ray-serve.single_deployment_dag:DagNode
+      route_prefix: /basic
+      runtime_env:
+        working_dir: "https://github.com/ray-project/kuberay/archive/master.zip"
+      deployments:
+        - name: BaseService
+          num_replicas: 1
+          max_replicas_per_node: 1
+          ray_actor_options:
+            num_cpus: 0.1
+```
+
+Look at the head Pod configuration `rayClusterConfig:headGroupSpec` embedded in the RayService YAML.  
+The configuration sets the CPU resources for the head Pod to 0 by passing the option `num-cpus: "0"` to `rayStartParams`. This setup avoids Ray Serve replicas running on the head Pod
+See [rayStartParams](https://github.com/ray-project/kuberay/blob/master/docs/guidance/rayStartParams.md) for more details.  
+```sh
+headGroupSpec:
+  rayStartParams:
+    num-cpus: "0"
+  template: ...
+```
+
+## Step 4: Verify the Kubernetes cluster status
+
+```sh
+# Step 4.1: List all RayService custom resources in the `default` namespace.
+kubectl get rayservice
+
+# [Example output]
+# NAME                              SERVICE STATUS   NUM SERVE ENDPOINTS
+# rayservice-no-ray-serve-replica   Running          2
+
+# Step 4.2: List all RayCluster custom resources in the `default` namespace.
+kubectl get raycluster
+
+# [Example output]
+# NAME                                               DESIRED WORKERS   AVAILABLE WORKERS   CPUS   MEMORY   GPUS   STATUS   AGE
+# rayservice-no-ray-serve-replica-raycluster-dnm28   2                 2                   1      6Gi      0               87s
+
+# Step 4.3: List services in the `default` namespace.
+kubectl get services
+
+# [Example output]
+# NAME                                                 TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)                                         AGE
+# ...
+# ice-no-ray-serve-replica-raycluster-dnm28-head-svc   ClusterIP   None          <none>        10001/TCP,8265/TCP,6379/TCP,8080/TCP,8000/TCP   3m22s
+# rayservice-no-ray-serve-replica-head-svc             ClusterIP   None          <none>        10001/TCP,8265/TCP,6379/TCP,8080/TCP,8000/TCP   3m1s
+# rayservice-no-ray-serve-replica-serve-svc            ClusterIP   10.96.8.250   <none>        8000/TCP                                        3m1s
+```
+
+## Step 5: Why 1 worker Pod isn't ready?
+
+```sh
```

Comment:
Add something similar to Step 4.1 in https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/rayservice-high-availability.html#step-4-verify-the-kubernetes-serve-service